package.json

```diff
@@ -8,9 +8,11 @@
     "test": "npm run test-node",
     "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
     "test-karma": "karma start karma.conf.js",
-    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "test-watch": "cross-env NODE_ENV=test mocha -r esm --watch --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text npm run test-node",
     "coverage-report": "nyc report",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "deleteCache": "rm -rf ./node_modules/.cache"
```

Comment:
What is this for?  Just on principle, I'd suggest avoiding such removal commands in script files.  They are scary, need to be very carefully checked, and here it looks like it could have side effects with other packages that use that standard dir.

Reply:
istanbul/ nyc has some issues with re-generating new test reports that is solved by deleting `node_modules/.cache` however I am removing this as it's not really necessary and could lead to confusion: https://github.com/digitalbazaar/edv-client/pull/26/commits/385f416292463b2731ae10291fabedc20764a945